### PR TITLE
B2 json from json or throw

### PR DESCRIPTION
**Pull request by spartan218**
Reminder that bzJson variable in B2JsonTest was renamed to b2Json.

### CODE REVIEW

**Comment by spartan218** [comment]
Not sure why the version file was changed marked as changed, nothing changed.

File: core/src/test/java/com/backblaze/b2/json/B2JsonTest.java [hunk 1]
```diff
@@ -1943,7 +1943,78 @@ public VersionedContainer(int x, int v) {
...
+    @Test
+    public void testToJson() {
+        final TestClassOne testClassOne = new TestClassOne("testABC", 1);
+        final String testClassOneStr = B2Json.toJsonOrThrowRuntime(testClassOne);
+        assertEquals("{\n  \"name\": \"testABC\",\n  \"number\": 1\n}", testClassOneStr);
+    }
+
+    @Test
+    public void testToJsonWithOptions() {
+        final B2JsonOptions options = B2JsonOptions.builder().build();
```
Comment (certainmagic) [inline core/src/test/java/com/backblaze/b2/json/B2JsonTest.java]: might be useful to test with a non-default option.

**Comment by spartan218** [comment]
@certainmagic added a test with a version number difference in the options, let me know if thats what you were thinking.

**Comment by certainmagic** [comment]
yep. thanks, briang.  :)

my goal was to just have something that showed the options were passed through in the one that took both arguments. i don't think we need both a default-options and non-default-options test.   but this is fine.  :)

**Pull request merged**